src/mlpack/methods/ann/layer/log_softmax_impl.hpp

```diff
@@ -64,6 +64,14 @@ LogSoftMaxType<MatType>::operator=(LogSoftMaxType&& other)
 
 template<typename MatType>
 void LogSoftMaxType<MatType>::Forward(const MatType& input, MatType& output)
+{
+  ForwardImpl(input, output);
+}
+
+template<typename MatType>
+void LogSoftMaxType<MatType>::ForwardImpl(const MatType& input, MatType& output,
+                                          const typename std::enable_if_t<
+                                          arma::is_arma_type<MatType>::value>*)
```

Comment:
```suggestion
void LogSoftMaxType<MatType>::ForwardImpl(const MatType& input,
                                          MatType& output,
                                          const typename std::enable_if_t<
                                              arma::is_arma_type<MatType>::value>*)
```
The usual style suggestions :smile: sorry if they can be frustrating at times, I'm just hoping to make things look consistent throughout.